### PR TITLE
Slovak Translations

### PR DESCRIPTION
**Pull request by reneklacan**
It definitely contains some typos. I will double check all translations tomorrow. But I also plan to start using FL in Slovak as soon as this is deployed and I will fix any typos as I find them.

### CODE REVIEW

File: sk.yml [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+  following:
+    follow: Sledovať
+    following: Sledovaní
+  footer:
+    about: O stránke
+    help: Pomoc
+  forgot_password:
+    email:
+      cta: Pošlať email s inštrukciami
+      error_forgot_email: I think you forgot something… your email address! :-p
```
Comment (rakvat) [inline sk.yml]: this and the next 2 lines are not translated

File: sk.yml [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+    sub_header: Nemýliť so Sparťanmi.
+    title: My. Sme. FetLife!
+  ad_stats:
+    actions:
+      delete: Vymazať
+      duplicate: Zduplikovať
+      edit: Upraviť
+      pause: Pozastaviť
+      start: Spustiť
+      unpause: Unpause
```
Comment (rakvat) [inline sk.yml]: unpause is the same?

File: sk.yml [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+      activity_feed: Novinky
+      activity_feed_short: Novinky
+      fp: Fresh<span class="a-ampersand">&amp;</span>Pervy
+      fp_short: Fresh
+      kp: Kinky<span class="a-ampersand">&amp;</span>Popular
+      kp_short: Popular
+    profile:
+      about: O mne
+      about_short: O mne
+      latest: Latest Activity
```
Comment (rakvat) [inline sk.yml]: not translated on purpose?

**Comment by reneklacan** [comment]
@rakvat All your comments are valid, I somehow missed those! Thank you!

**Comment by johnbaku** [comment]
@reneklacan wow... money... that was quick... you hire someone to do that?!?! :-p

**Comment by reneklacan** [comment]
@johnbaku then it's good that you don't mind!

**Pull request merged**